apps/style/netsim/style.scss

```diff
@@ -24,7 +24,6 @@ $wide-column-width: 121px;
   user-select: $val;
 }
 
-// Reduce top margin to help fit everything above the fold
 .full_container {
   padding-top: 2px;
```

Comment:
not explicitly related to your diff, but you may want to remove this. (1) from a component perspective, you shouldn't really know the classes used by the outer frame (dashboard), (2) the default padding-top for full_container is now 0px, so I think you are actually increasing it with these 2px.